cmd/aks-mcp/container_test.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"github.com/Azure/aks-mcp/internal/config"
+	"github.com/Azure/aks-mcp/internal/server"
+	"net/http"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+)
+
+// TestContainerTransportConfiguration validates that the transport configuration
+// is appropriate for container deployment
+func TestContainerTransportConfiguration(t *testing.T) {
```

Comment:
the test here is running locally instead of inside container. If the goal is test container behavior, then you should build the image, start the container and then validate container configurations and APIs